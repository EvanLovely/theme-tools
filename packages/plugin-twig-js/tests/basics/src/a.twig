{% set text = 'Hello World' %}
<h1>{{ text }}</h1>
