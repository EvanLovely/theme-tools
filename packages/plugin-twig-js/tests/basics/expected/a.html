<h1>Hello World</h1>

<h2>Here's a list generated from data.json with twig.js for each</h2>
<ul>
            <li>one</li>
            <li>two</li>
            <li>three</li>
    </ul>

<h3>A number: 42</h3>

<h3>A string: Hello World</h3>